File: Holiday Assignments/Java/PizzaWahala.java
import java.util.Scanner;

public class PizzaWahala {
    public static void main(String[] args) {
        Scanner userInput = new Scanner(System.in);
        System.out.print("Number of people = ");
        int numberOfPeople = userInput.nextInt();
        userInput.nextLine();
	if (numberOfPeople <= 0){
        System.out.println("Invalid Input");
    } else {
	System.out.print(
    """
    Pizza type \t No. of slices \t Prices
    1. Sapa Size \t 4 slices \t 2000
    2. Small Money \t 6 slices \t 2400
    3. Big Boys  \t 8 slices \t 3000
    4. Odogwu \t 12 slices \t 4200
    """);

        System.out.print("Piza type = ");
        String pizzaType = userInput.nextLine().toLowerCase();
        System.out.println(pizzaType);

        int numberOfSlices = 0;
        int pricePerBox = 0;
        int numberOfBoxes = 0;
        int leftOver = 0;
        int price = 0;

            switch (pizzaType) {
                case "sapa size":
                    numberOfSlices = 4;
                    pricePerBox = 2000;

                case "small money":
                    numberOfSlices = 6;
                    pricePerBox = 2400;

                case "big boys":
                    numberOfSlices = 8;
                    pricePerBox = 3000;

                case "odogwu":
                    numberOfSlices = 12;
                    pricePerBox = 4200;
    
                default:
                    System.out.println("Invalid Input");
                    break;
                
                    
            }
        numberOfBoxes = (numberOfPeople + numberOfSlices - 1) / numberOfSlices;
        leftOver = (numberOfBoxes * numberOfSlices) - numberOfPeople;
        price = pricePerBox * numberOfBoxes;

        System.out.println("Number of boxes of pizza to buy = " + numberOfBoxes + " boxes");
        System.out.println("Number left over slice after serving = " + leftOver + " slices");
        System.out.println("Prices = " + price);	
	    }
    }
}
        

File: Holiday Assignments/Python/pizza_wahala.py
def pizza_wahala():
    number_of_people = int(input("Number of people = "))
    if number_of_people <= 0:
        print("Invalid Input")
        
    else:
        print("""
    Pizza type \t    No. of slices \t Prices
    1. Sapa Size \t 4 slices \t 2000
    2. Small Money \t 6 slices \t 2400
    3. Big Boys \t 8 slices \t 3000
    4. Odogwu   \t 12 slices \t 4200
    """)

        pizza_type = input("Piza type = ").lower()
        match pizza_type:
                case "sapa size":
                    number_of_slices = 4
                    price_per_box = 2000

                case "small money":
                    number_of_slices = 6
                    price_per_box = 2400

                case "big boys":
                    number_of_slices = 8
                    price_per_box = 3000

                case "odogwu":
                    number_of_slices = 12
                    price_per_box = 4200  

                case _:
                    print ("Invalid Input")
                    return

        number_of_boxes = (number_of_people + number_of_slices - 1) // number_of_slices
        left_over = (number_of_boxes * number_of_slices) - number_of_people 
        price = price_per_box * number_of_boxes 
        print("Number of boxes of piza to buy = ", number_of_boxes, " boxes", sep = "") 
        print("Number left over slice after serving = ", left_over ," slices", sep = "")
        print("Prices = ", price)

pizza_wahala()